modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -136,6 +136,9 @@
 							data[index] = null;
 						}
 					});
+					thumbCuePoint = thumbCuePoint.filter(function(cuepoint) {
+						return (typeof(cuepoint.assetId) !== 'undefined');
+					});
 					$.each(thumbCuePoint, function (index, item) {
```

Comment:
need to check where we get the thumbnail url if there's no asset id, we will need to add the referrer there as well.